File: sandbox/dotnet/src/Dropbox.SignSandbox/SignatureRequestFilesAsFileUrlDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class SignatureRequestFilesAsFileUrlDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();

        try
        {
            var response = new SignatureRequestApi(config).SignatureRequestFilesAsFileUrl(
                signatureRequestId: "fa5c8a0b0f492d768749333ad6fcc214c111e967",
                forceDownload: 1
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling SignatureRequest#SignatureRequestFilesAsFileUrl: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: sandbox/ruby/src/SignatureRequestFilesAsFileUrlDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

begin
    response = Dropbox::Sign::SignatureRequestApi.new.signature_request_files_as_file_url(
        "fa5c8a0b0f492d768749333ad6fcc214c111e967",
        {
            force_download: 1,
        },
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling SignatureRequest#signature_request_files_as_file_url: #{e}"
end
